integration/app/cache_test.go

```diff
@@ -0,0 +1,110 @@
+package app_test
+
+import (
+	"context"
```

Comment:
format imports

Reply:
Does the linter not catch these? I'm used to just fixing these with the `make lint` command, but it didn't complain on these?

Reply:
I think it doesn't unfortunately. So we are filtering this on a review.